known-imports.yaml

```diff
@@ -0,0 +1,5 @@
+imports:
```

Comment:
It looks like known-imports isn't kicking in without a value for the `imports` key eg
```
imports: {}
```